docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,25 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+OpenShift provides an integrated docker registry which can be used as a local registry.
+
+This helps to speed up the local development as the images present in the registry can be used as imagestream for running an application in the OpenShift
+
+. Start minishift with the link:../command-ref/minishift_start{outfilesuffix}[`minishift start`] command.
+
+. Add the oc binary to the PATH environment variable.
+
+ $ export PATH=$PATH:~/.minishift/cache/oc/v1.5.0-rc.0
```

Comment:
don't hard code the oc version. use a variable instead, see interacting-with-openshift.adoc in the OpenShift client library section.